content/docs/get-started/kubernetes/modify-program.md

```diff
@@ -135,6 +135,109 @@ else:
     pulumi.export("ip", frontend.status.apply(lambda v: v["load_balancer"]["ingress"][0]["ip"] if "load_balancer" in v else None))
 ```
 
+```csharp
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Pulumi;
+using Pulumi.Kubernetes.Core.V1;
+using Pulumi.Kubernetes.Apps.V1;
+using Pulumi.Kubernetes.Types.Inputs.Core.V1;
+using Pulumi.Kubernetes.Types.Inputs.Apps.V1;
+using Pulumi.Kubernetes.Types.Inputs.Meta.V1;
+
+class Program
+{
+    static Task<int> Main()
+    {
+        return Pulumi.Deployment.RunAsync(() =>
+        {
+            var config = new Pulumi.Config();
+            var isMinikube = config.GetBoolean("isMinikube") ?? false;
+
+            var appName = "nginx";
+
+            var appLabels = new InputMap<string>{
+                { "app", appName },
+            };
+
+            var deployment = new Pulumi.Kubernetes.Apps.V1.Deployment(appName, new DeploymentArgs
+            {
+                Spec = new DeploymentSpecArgs
+                {
+                    Selector = new LabelSelectorArgs
+                    {
+                        MatchLabels = appLabels,
+                    },
+                    Replicas = 1,
+                    Template = new PodTemplateSpecArgs
+                    {
+                        Metadata = new ObjectMetaArgs
+                        {
+                            Labels = appLabels,
+                        },
```

Comment:
One-liners for single-property definition should fit the string and make it shorter in height

Reply:
Going to leave as-is for now - as horizontal space is pretty tight already for these examples - and the current format is consistent with many other examples we have (and with other parts of the same example).  Though open to adopting this pattern for single property definitions broadly in follow-up if that's what you think is best to standardize on.